app/controllers/concerns/idv/ab_test_analytics_concern.rb

```diff
@@ -4,7 +4,12 @@ module AbTestAnalyticsConcern
     include Idv::GettingStartedAbTestConcern
 
     def ab_test_analytics_buckets
-      acuant_sdk_ab_test_analytics_args.
+      buckets = {}
+      if defined?(idv_session)
```

Comment:
I think we can avoid some of the footwork here with this:

```ruby
user_session&.dig(:idv, :skip_hybrid_handoff)
```

That is almost an equivalent implementation. The above will fetch a value if it exists even if `idv_session` is not defined.

Reply:
When I made this change, in specs I got "Warden::NotAuthenticated: :user user is not logged in" when I tried to access user_session without a signed-in user.